ReSharper.FSharp/src/FSharp.Psi/src/Impl/Tree/TopReferencePat.cs

```diff
@@ -18,5 +18,7 @@ internal partial class TopReferencePat
     public TreeNodeCollection<IAttribute> Attributes =>
       this.GetBindingFromHeadPattern()?.Attributes ??
       TreeNodeCollection<IAttribute>.Empty;
+
+    public override XmlDocBlock XmlDocBlock => Parent?.FirstChild as XmlDocBlock;
```

Comment:
Likely not a very common issue here, but something like this won't work:
```fsharp
/// X
let (x) = 1
```
Maybe use typed APIs like `BindingNavigator.GetByHeadPattern(this.IgnoreParentParens())` to get the declaration instead?